openml/extensions/sklearn/extension.py

```diff
@@ -1531,15 +1534,31 @@ def _prediction_to_probabilities(y: np.ndarray, classes: List[Any]) -> np.ndarra
             -------
             np.ndarray
             """
+
+            if isinstance(task, (OpenMLClassificationTask, OpenMLLearningCurveTask)):
+                print("\n\nInside _prediction_to_probabilities\n\n")
```

Comment:
Don't leave print statements in the code. Use `logging` module.

Reply:
Thanks for catching that @PGijsbers. Actually, please don't log here at all. @PGijsbers can you think of a reason to use a print statement in regular code? Maybe that can be checked by black, too?

Reply:
I don't Black will do that, Black only formats and checks that the resulting syntax trees are identical (if I remember the right terminology). They wouldn't be if expressions are removed.
However I did find a Flake8 plugin [flake8-print](https://pypi.org/project/flake8-print/) which will warn you about print statements. It's not automatic removal - but that's probably a good thing considering in the worst case you call a function with side effects from within your print statement, and the print statement needs to be removed with some care.

Reply:
That sounds great to have.